### DISCUSSION TITLE
Is there a way to adjust the size of each window?

### DISCUSSION THREAD

**Discussion by lucharo** [tconbeer/harlequin#693] [Ideas (Feature Requests)]
As the title goes, I'd like to make the section on the left wider

**Comment by tconbeer** [comment]
Not today.

**Reply by lucharo** [reply]
Where is the size of each window set? Collapsing the side bar could be very useful as well, but maybe that's asking too much out of a TUI

**Reply by tconbeer** [reply]
You can collapse the side bar with ctrl b or go full screen with f10
